From 4e5509411cefa4f8a48a48160ab4215a093ba7c7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
 <frederic.epitre@orange.fr>
Date: Fri, 19 Oct 2018 08:02:12 +0200
Subject: [PATCH] anaconda: check/add user to 'wheel' and 'qubes' groups
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Originally from 378cfc44dd218c61b838f4f9011dcfd790df59eb by Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>

Signed-off-by: Frédéric Pierret (fepitre) <frederic.epitre@orange.fr>
---
 pyanaconda/ui/gui/spokes/user.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/pyanaconda/ui/gui/spokes/user.py b/pyanaconda/ui/gui/spokes/user.py
index 623525a7b..c25150272 100644
--- a/pyanaconda/ui/gui/spokes/user.py
+++ b/pyanaconda/ui/gui/spokes/user.py
@@ -386,6 +386,11 @@ class UserSpoke(NormalSpoke, GUISpokeInputCheckHandler):
 
         self._user.name = self.username
 
+        if "wheel" not in self._user.groups:
+            self._user.groups.append("wheel")
+        if "qubes" not in self._user.groups:
+            self._user.groups.append("qubes")
+
         # Copy the spoke data back to kickstart
         # If the user name is not set, no user will be created.
         if self._user.name:
-- 
2.20.1

